ubuntu-server.js

```diff
@@ -0,0 +1,122 @@
+#!/usr/bin/env node
+
+/**
+ * Copyright (C) 2016, Canonical Ltd.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ */
+
+var net = require('net');
+var repl = require('repl');
+var vm = require('vm');
+var util = require('util');
+var Buffer = require('buffer').Buffer;
+var realmConstructor = require("./node_modules/realm/lib/index.js");
+console.log("Loaded realmConstructor: " + realmConstructor);
+
+var DEBUG = 1;
+
+function rnUbuntuServer(readable, writable) {
+  console.reportErrorsAsExceptions = false;  // XXX:
+  var sandbox = { console: console, util: util, outerRealmConstructor: realmConstructor };
```

Comment:
`realmConstructor` is passed as `outerRealmConstructor` global variable into sandoxed env